### ISSUE TITLE
More serious warp effects happen than allowed by current warp level

### ISSUE THREAD

**Issue by ZuOverture** [GTNewHorizons/GT-New-Horizons-Modpack#9193]
### Your GTNH Discord Username

AnCorp#8466

### Your Pack Version

2.1.2.0

### Your Server

Epsilon

### Your Expectation

My current warp is as follows:
Permanent=21, Normal=9, Temporary=0
So total value is 30. According to the table located [here](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/8611#issuecomment-958003423) I should only see effects from 1-50 group up to those having 30 in the right column (warp amount). I shouldn't ever get the Jump Boost(45), Blood(35), Friendly Creepers(40) or Nausea(45).

### The Reality

 I still get the Jump Boost, Blood Rain, Friendly Creepers or Nausea periodically.

### Your Proposal

a) remove the right column completely, since warp range(1-50) is the only thing that currently matters
or
b) make every effect happen only after its stated warp level

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [ ] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by bombcar** [comment]
There is a warptheory.cfg which shows the levels - also there are some that are from thaum itself and can’t be changed. We can edit the wiki

**Comment by ZuOverture** [comment]
Values in my local nonmodified warptheory.cfg correspond with those in table, for what its worth.
Thinking about it, if "Your veins catch fire" means Poison effect, then I can receive lvl65 effect too.

**Comment by bombcar** [comment]
Can you try the InGameInfo xml tweak on that same wiki page and confirm it reports the same amounts?

**Comment by ZuOverture** [comment]
Measured warp levels using arcane litmus, is that acceptable?

**Comment by bombcar** [comment]
That should be the same - the igi is just continuous. It’s the sum of all.

**Comment by D-Cysteine** [comment]
Do you ever wear any armor with the `Warping` effect?

**Comment by ZuOverture** [comment]
> Do you ever wear any armor with the `Warping` effect?
No.

**Comment by D-Cysteine** [comment]
Could you post the entirety of your Warp Theory config file?

It does look like the effects that you are seeing that should not be occurring, are all listed as under 30 warp according to the original warp table:
 * https://gtnh.miraheze.org/wiki/Thaumcraft_Tips

so I wonder if your game might somehow still be using the old values. Have you seen any of the brand-new warp effects (not present in the old table), to confirm that you're using the new version?

**Comment by ZuOverture** [comment]
My observations were made on Epsilon. Don't have access to config obviously.

**Comment by boubou19** [comment]
My fault, didn't updated the enforced config for the warp theory mod. Please check after the next reboot.

**Comment by boubou19** [comment]
Seems to be fixed.

**Comment by ZuOverture** [comment]
Looks like it's not fixed yet. I've seen effects like Blood etc on the next day after boubou's replaced the config. Took some time to make a flux scrubber focus to reduce my warp a bit (now at 21 permawarp), but I still get Jump Boost(45) eventually (but no Lightning or Blood anymore).

**Comment by bombcar** [comment]
It counts all warp not just perma - the ingameinfo config that shows warp may be useful

**Comment by ZuOverture** [comment]
I know that, and only permawarp is nonzero. Probably should've explicitly said that.

**Comment by ZuOverture** [comment]
Please reopen, it either wasn't fixed, or broken again.

**Comment by boubou19** [comment]
should be fixed now, will reopen it if players tell me it's not actually fixed

**Comment by PeanutbutterWarrior** [comment]
I think there's been a regression on this. I'm playing on the delta server and I'm getting much higher effects than I should. I've got 70 permenant warp and 31 normal warp, and I'm getting spawn wither (200)

**Comment by sisyphussy** [comment]
are you sure it was a real wither and not the fake jumpscare one?

**Comment by PeanutbutterWarrior** [comment]
Definitely not, it did the lightning from being inside a claim, and when the wither spawned it killed me. Additionally, the fake wither needs 150 warp

**Comment by sisyphussy** [comment]
make a new issue and i'll look into it.

**Comment by sisyphussy** [comment]
@PeanutbutterWarrior i couldn't reproduce unfortunately.
